### PR TITLE
fix(issue-template): update link to GitHub Discussions

### PR DESCRIPTION
**Pull request by rakleed**
Beacuse of https://github.com/mdn/yari/discussions/6285

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -124,5 +124,5 @@ body:
           required: true
         - label: I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
           required: true
-        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).
+        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/mdn-community/discussions).
```
Comment (caugner) [inline .github/ISSUE_TEMPLATE/bug-report.yml:127]: Let's link to the org discussions instead:

```suggestion
        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions).
```

**Review by caugner — CHANGES_REQUESTED** [review]

**Review by caugner — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -124,5 +124,5 @@ body:
           required: true
         - label: I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
           required: true
-        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).
+        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions).
```
Comment (caugner) [inline .github/ISSUE_TEMPLATE/bug-report.yml:127]: @schalkneethling Should we link here specifically to the [platform section](https://github.com/orgs/mdn/discussions/categories/platform)?

Comment (schalkneethling) [reply .github/ISSUE_TEMPLATE/bug-report.yml:127]: > @schalkneethling Should we link here specifically to the [platform section](https://github.com/orgs/mdn/discussions/categories/platform)?

Yeah, that sounds like a good idea. Thank you, @rakleed and @caugner 🎉

Comment (caugner) [reply .github/ISSUE_TEMPLATE/bug-report.yml:127]:
```suggestion
        - label: I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).
```

**Comment by schalkneethling** [comment]
Welcome to the community, @rakleed  🎉 Thank you for your contribution. If you are interested in more ways to contribute to MDN Web Docs, there is a [contributor task board](https://github.com/orgs/mdn/projects/25). If you have any questions or need help finding something to work on, feel free to ask for help in our [community discussions](https://github.com/orgs/mdn/discussions). We look forward to more of your contributions. 🙌

**Pull request merged**